project.py

```diff
@@ -0,0 +1,153 @@
+from bs4 import BeautifulSoup
+import requests
+import re
+import pickle
+import os.path
+from pathlib import Path
+import random
+
+def dict_creation_one(script, things = dict()):
+    '''Create the base dictionary for ONE word; only to be called if there is no dicts1.p already'''
+    entiredict =  things
+    everything = script
+    newstring = ''
+    for x in script: #get rid of symbols that I don't want in my speech
+        if x not in ('[', ']', '@', '#', '$', '%', '^', '&', '*', '(', ')', ':', '"', '\\', '—', '|', '-'):
+            newstring += (x)
+    newstring = newstring.lower()
+    listowords = newstring.split()
+    index = 0
+    for y in listowords:    #run through all of the words and create a dictionary with them
+        index += 1
+        if index != len(listowords):
+            if y not in entiredict:
+                entiredict[y] = [listowords[index]]
+            else:
+                entiredict[y].append(listowords[index])
+    return entiredict
+
+def dict_creation_two(script, things = dict()):
+    '''Create the base dictionary for TWO words; only to be called if there is no dicts.p already'''
+    entiredict =  things
+    everything = script
+    newstring = ''
+    for x in script: #get rid of symbols that I don't want in my speech
```

Comment:
You have this thing twice; I would write a separate function for it (so it's more readable) or pass in a pre-cleaned script